src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -575,33 +575,14 @@ private void ScanInterpolatedStringLiteralHoleBalancedText(char endingChar, bool
                             ScanInterpolatedStringLiteralNestedString();
                             continue;
                         case '@':
-                            if (_lexer.TextWindow.PeekChar(1) == '"' && !RecoveringFromRunawayLexing())
                             {
-                                // check for verbatim string inside an expression hole.
-                                var nestedStringPosition = _lexer.TextWindow.Position;
-
-                                // Note that this verbatim string may encounter an error (for example if it contains a
-                                // new line and we don't allow that).  This should be reported to the user, but should
-                                // not put us into an unrecoverable position.  We can clearly see that this was intended
-                                // to be a normal verbatim string, so we can continue on an attempt to understand the
-                                // outer interpolated string properly.
                                 var discarded = default(TokenInfo);
-                                var errorCode = _lexer.ScanVerbatimStringLiteral(ref discarded);
-                                if (errorCode is ErrorCode code)
-                                {
-                                    TrySetRecoverableError(_lexer.MakeError(nestedStringPosition, width: 2, code));
```

Comment:
Is `TrySetRecoverableError` still used?

Reply:
discussed offline.

Reply:
Looking at the code after our offline discussion, it looks like `TrySetRecoverableError` is unused. We could remove that method.

Reply:
agreed.  i can do that in a followup.